src/main/java/appeng/core/sync/network/AppEngClientPacketHandler.java

```diff
@@ -1,69 +1,69 @@
- */
-
-package appeng.core.sync.network;
-
-
-import java.lang.reflect.InvocationTargetException;
-
-import io.netty.buffer.ByteBuf;
-
-import net.minecraft.client.Minecraft;
-import net.minecraft.entity.player.EntityPlayer;
-
-import cpw.mods.fml.common.network.internal.FMLProxyPacket;
-
-import appeng.core.AELog;
-import appeng.core.sync.AppEngPacket;
-import appeng.core.sync.AppEngPacketHandlerBase;
-
-
-public class AppEngClientPacketHandler extends AppEngPacketHandlerBase implements IPacketHandler
-{
-
-	@Override
-	public void onPacketData( final INetworkInfo network, final FMLProxyPacket packet, EntityPlayer player )
-	{
-		final ByteBuf stream = packet.payload();
-
-		player = Minecraft.getMinecraft().thePlayer;
-
-		try
-		{
-			final int packetType = stream.readInt();
-			final AppEngPacket pack = PacketTypes.getPacket( packetType ).parsePacket( stream );
-			pack.clientPacketData( network, pack, player );
-		}
-		catch( final InstantiationException e )
-		{
-			AELog.error( e );
-		}
-		catch( final IllegalAccessException e )
-		{
-			AELog.error( e );
-		}
-		catch( final IllegalArgumentException e )
-		{
-			AELog.error( e );
-		}
-		catch( final InvocationTargetException e )
-		{
-			AELog.error( e );
-		}
-	}
-}
+///*
+// * This file is part of Applied Energistics 2.
+// * Copyright (c) 2013 - 2014, AlgorithmX2, All rights reserved.
+// *
+// * Applied Energistics 2 is free software: you can redistribute it and/or modify
+// * it under the terms of the GNU Lesser General Public License as published by
+// * the Free Software Foundation, either version 3 of the License, or
+// * (at your option) any later version.
+// *
+// * Applied Energistics 2 is distributed in the hope that it will be useful,
+// * but WITHOUT ANY WARRANTY; without even the implied warranty of
+// * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// * GNU Lesser General Public License for more details.
+// *
+// * You should have received a copy of the GNU Lesser General Public License
+// * along with Applied Energistics 2.  If not, see <http://www.gnu.org/licenses/lgpl>.
+// */
+//
+//package appeng.core.sync.network;
+//
+//
+//import java.lang.reflect.InvocationTargetException;
+//
+//import io.netty.buffer.ByteBuf;
+//
+//import net.minecraft.client.Minecraft;
+//import net.minecraft.entity.player.EntityPlayer;
+//
+//import cpw.mods.fml.common.network.internal.FMLProxyPacket;
+//
+//import appeng.core.AELog;
+//import appeng.core.sync.AppEngPacket;
+//import appeng.core.sync.AppEngPacketHandlerBase;
+//
+//
+//public class AppEngClientPacketHandler extends AppEngPacketHandlerBase implements IPacketHandler
+//{
+//
+//	@Override
+//	public void onPacketData( final INetworkInfo network, final FMLProxyPacket packet, EntityPlayer player )
+//	{
+//		final ByteBuf stream = packet.payload();
+//
+//		player = Minecraft.getMinecraft().thePlayer;
+//
+//		try
+//		{
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)